webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/lift/TechniqueApi.scala

```diff
@@ -335,9 +335,14 @@ class TechniqueApi (
         } )
     }
 
-    private def isTechniqueNameExist(bundleName: BundleName) = {
+    private def isTechniqueNameExist(techniqueName: String) = {
       val techniques = techniqueRepository.getAll()
-      techniques.keySet.map(_.name.value).contains(bundleName.value)
+      techniques.values.map(_.name.toLowerCase).toList.contains(techniqueName.toLowerCase)
```

Comment:
why do you need the toList ?

Reply:
`techniques.values` produce `Iterable[Technique]` and `contains` do not apply on `Iterable[A]`

Reply:
you should advertise in the comment (at least) that it's a case insensitive comparision